packages/design-system-react/package.json

```
@@ -82,7 +82,7 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "@metamask/design-system-tailwind-preset": "^0.5.0",
+    "@metamask/design-system-tailwind-preset": "^0.0.0",
```

This should match `@metamask/design-system-tailwind-preset` bumped version
```suggestion
    "@metamask/design-system-tailwind-preset": "^0.6.0",
```

---

Updated https://github.com/MetaMask/metamask-design-system/pull/784/commits/c0a41ed17893e82b1657a2fc5573e5a1acda8c68